scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -327,3 +327,15 @@ def test_equal_area_regions(self, poly):
         sv = SphericalVoronoi(points)
         areas = sv.calculate_areas()
         assert_almost_equal(areas, 4 * np.pi / len(points))
+
+    @pytest.mark.parametrize("radius", [0.5, 1, 2])
+    @pytest.mark.parametrize("center", [(0, 0), (1, 2)])
+    def test_area_reconstitution_2d(self, radius, center):
+        np.random.seed(0)
+        points = np.random.normal(size=(100, 2))
+        points /= np.linalg.norm(points, axis=1)[:, None]
+        sv = SphericalVoronoi(radius * points + center,
+                              radius=radius,
+                              center=center)
+        areas = sv.calculate_areas()
+        assert_almost_equal(areas.sum(), 2 * np.pi * radius)
```

Comment:
Other ideas for tests might be:

1) Equally spaced points (i.e., slerped) on the circle should produce Voronoi regions with identical areas.
2) Doubling the density of the equally spaced points should halve the region areas.

Reply:
I have added the 1st test, for n=10, 100, and 1000. The test verifies that the individual areas are correct, so I think this encompasses the scope of the 2nd test.